backend/core/src/listings/listings.service.ts

```diff
@@ -121,6 +122,20 @@ export class ListingsService {
     })
     listingDto.unitsAvailable = availableUnits
 
+    if (
+      listing.status === ListingStatus.active &&
+      listingDto.applicationDueDate &&
+      dayjs(listingDto.applicationDueDate).isBefore(new Date())
+    ) {
+      listingDto.status = ListingStatus.closed
+    } else if (
+      listing.status === ListingStatus.closed &&
+      listingDto.applicationDueDate &&
+      dayjs(listingDto.applicationDueDate).isAfter(new Date())
+    ) {
+      listingDto.status = ListingStatus.active
+    }
```

Comment:
This is causing a bug. when you "post results".
Steps to reproduce:
1. Select a listing and manually close it.
2. Click "Edit" on that property and click "Post Results". 
3. After saving the PDF the property is now listed as "open".

I'm wondering if it was the intention of this change that on save of a listing that was manually closed before the application due date we would automatically open it back up

Reply:
@YazeedLoonat do you have context here?

Reply:
@emilyjablonski @ludtkemorgan so this change came into existence because of the issue where if a listing should have been closed (when looking at the application due date the listings weren't actually getting flagged as closed

since that issue isn't directly tied to duplicates I'd feel comfortable walking these changes back and then looking to resolve that in a stand alone issue

@seanmalbert thoughts?

Reply:
This has been pulled out.